firefox-ios/Client/Frontend/Browser/Tabs/Views/TabTrayViewController.swift

```diff
@@ -460,6 +463,24 @@ class TabTrayViewController: UIViewController,
         store.dispatch(TabPanelAction.closeAllTabs(windowUUID.context))
     }
 
+    private func showCloseAllConfirmation() {
+        let controller = AlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        controller.addAction(UIAlertAction(title: .AppMenu.AppMenuCloseAllTabsTitleString,
+                                           style: .default,
+                                           handler: { _ in self.confirmCloseAll() }),
+                             accessibilityIdentifier: AccessibilityIdentifiers.TabTray.deleteCloseAllButton)
+        controller.addAction(UIAlertAction(title: .TabTrayCloseAllTabsPromptCancel,
+                                           style: .cancel,
+                                           handler: nil),
+                             accessibilityIdentifier: AccessibilityIdentifiers.TabTray.deleteCancelButton)
+        controller.popoverPresentationController?.barButtonItem = deleteButton
+        present(controller, animated: true, completion: nil)
+    }
```

Comment:
Just curious, we don't have any existing Alert Controller designs for destructive actions? The UX of this to me doesn't emphasize the fact that this is to delete a bunch of data (for example, using red in the confirmation)

Reply:
Improving the UX is mostly out of scope for this project so I think it's fine to keep it the same for now but this is a very good point, would you mind creating a bug for it and we can tackle it through our regular triage process since it' needs UX input.

Reply:
Created a bug here https://mozilla-hub.atlassian.net/browse/FXIOS-8399